### PR TITLE
Update Korean translation (lines 1501-2000) (#7427)

### PR DESCRIPTION
**Pull request by Seo-yul**
Issues: https://github.com/kubernetes/dashboard/issues/7424
from: https://github.com/kubernetes/dashboard/issues/7424#issuecomment-1238113138
- Updated line number : 1501 - 2000
- Changed `<target state="new">` to `<target>`
- Translated strings to Korean, between `<target state="new">` Strings to translate `</target>`

### CODE REVIEW

File: i18n/ko/messages.ko.xlf [hunk 1]
```diff
@@ -1798,7 +1798,7 @@
       </trans-unit>
       <trans-unit id="2275326697204049263" datatype="html">
         <source><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> bytes</source>
-        <target state="new"><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> bytes</target>
+        <target><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> 바이트</target>
```
Comment (seokho-son) [inline i18n/ko/messages.ko.xlf:1801]:
```suggestion
        <target><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> bytes</target>
```

I'd like to leave the unit `bytes` in original letter.

File: i18n/ko/messages.ko.xlf [hunk 2]
```diff
@@ -1846,39 +1846,39 @@
...
+        <target>지원되지 않는 그래프 타입 <x id="PH" equiv-text="this.graphType"/>.</target>
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/graph/component.ts</context>
           <context context-type="linenumber">95</context>
         </context-group>
       </trans-unit>
       <trans-unit id="3804893288209000257" datatype="html">
         <source>Waiting for more data to display chart...</source>
-        <target state="new">Waiting for more data to display chart...</target>
+        <target>차트를 표시할 추가 데이터를 기다리는 중...</target>
```
Comment (seokho-son) [inline i18n/ko/messages.ko.xlf:1881]: How about this ?

```suggestion
        <target>차트를 표시하기 위해 추가 데이터를 기다리는 중...</target>
```

**Review by seokho-son — COMMENTED** [review]
Hi @Seo-yul 
Thank you for the contribution ! :)

I have minor comments. PTAL

File: i18n/ko/messages.ko.xlf (hunk 2 above)
Comment (Seo-yul) [reply i18n/ko/messages.ko.xlf]: I think it's more cool :D !

**Comment by seokho-son** [comment]
Hi @Seo-yul 
Thank you. The updated PR LGTM :)

Please squash commits.

**Comment by seokho-son** [comment]
Hi @shu-mutou
@Seo-yul is a first-time contributor of this repo.
Could you approve running workflows for this PR too ?

**Comment by Seo-yul** [comment]
> Hi @Seo-yul Thank you. The updated PR LGTM :)
> 
> Please squash commits.

I done. :D

**Pull request merged**